main/tests/server/src/com/google/refine/browsing/facets/ListFacetTests.java

```diff
@@ -99,9 +100,11 @@ public void serializeListFacetConfig() throws JsonParseException, JsonMappingExc
     
     @Test
     public void serializeListFacet() throws JsonParseException, JsonMappingException, IOException {
-        Project project = createCSVProject("Column A\n" +
-                "foo\n" +
-                "bar\n");
+        Project project = createProject(new String[]
```

Comment:
The `createCSVProject` method is one I introduced some years ago as a convenience method to quickly create a project in the context of a test. This had a couple of problems:
* internally, it relies on our CSV importer, which is not very clean: this is a rather complex component, which would get run by a lot of different tests, even though those corresponding tests have nothing to do with it (they test something unrelated).
* specifying a project in CSV form is not very readable in a Java source file, it forces the human reader to parse the CSV visually to understand the structure of the project being created
* creating a project from a CSV is not very flexible: for instance, it is complicated to specify a project with a mixture of empty strings and null values, or with parsed numbers and numbers as strings

So that's why I introduced this new method, which lets testers create projects without going through the CSV importer. The reason why this change is done in the same go as the Maven structure change is that this structure change prevents us from using the CSV importer in testing utilities, because the testing utilities module does not depend on the workflow elements module.